driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -173,6 +174,26 @@ public static Bson unwind(final String fieldName) {
         return new BsonDocument("$unwind", new BsonString(fieldName));
     }
 
+    /**
+     * Creates a $unwind pipeline stage for the specified field name, which must be prefixed by a {@code '$'} sign.
+     *
+     * @param fieldName the field name, prefixed by a {@code '$' sign}
+     * @param unwindOptions options for the unwind pipeline stage
+     * @return the $unwind pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/unwind/ $unwind
+     * @mongodb.server.release 3.2
```

Comment:
This implies to me that $unwind is server 3.2.  Instead, let's mark the methods in UnwindOptions as 3.2.